### PR TITLE
Add highlight group for email

### PR DESCRIPTION
**Pull request by alfunx**
Personally I like to highlight emails addresses in a different color to make them less visible. This adds a new highlight group for email addresses, which is by default linked to `gitmessengerPopupNormal` (to keep the default behavior unchanged).

### CODE REVIEW

**Comment by rhysd** [comment]
I agree. E-mail information is less valuable when looking into code changes.

File: syntax/gitmessengerpopup.vim [hunk 1]
```diff
@@ -5,6 +5,7 @@ endif
 syn match gitmessengerHeader '\_^ \%(History\|Commit\|Date\|Author\|Committer\):' display
 syn match gitmessengerHash '\%(\<Commit: \)\@<=[[:xdigit:]]\+' display
 syn match gitmessengerHistory '\%(\<History: \)\@<=#\d\+' display
+syn match gitmessengerEmail '\%(\<\(Author\|Committer\): .*\)\@<=<.\+>' display
```
Comment (rhysd) [inline syntax/gitmessengerpopup.vim]: Please add `\_^ ` to ensure this syntax highlighting matches to headers.

```suggestion
syn match gitmessengerEmail '\_^ \%(\%(Author\|Committer\): .*\)\@<=<.\+>' display
```

Comment (alfunx) [reply syntax/gitmessengerpopup.vim]: Actually I didn't intend to match the headers themselves here. The author line may look like this for example:

```
 Author: John Doe <john.doe@mail.com>
```

I just want to match `<john.doe@mail.com>` in this case. This is similar to matching the hash and the history number.

If we want to match `\_^ `, it should probably be done inside the parenthesis, for all cases (so for `gitmessengerHash` and `gitmessengerHistory` as well). I'll make another pull-request for that if you wish :slightly_smiling_face:

Comment (alfunx) [reply syntax/gitmessengerpopup.vim]: But I missed a `%`.

**Comment by rhysd** [comment]
Thanks for the updates. I merged this PR at first then added a small patch to match headers more exactly at f7d8f4e

**Comment by alfunx** [comment]
> added a small patch to match headers more exactly at f7d8f4e

Thanks, that's what I had ready as next PR! :slightly_smiling_face:

**Pull request merged**